ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -1388,6 +1400,16 @@ private class JdbcLedgerDao(
           |where package_id = {package_id}
           |""".stripMargin)
 
+  private val SQL_SELECT_PACKAGE_UPLOAD_ENTRIES =
+    SQL(
+      "select * from package_upload_entries where ledger_offset>={startInclusive} and ledger_offset<{endExclusive} order by ledger_offset asc")
+
+  private val SQL_SELECT_PACKAGE_UPLOAD_ENTRY =
+    SQL("select * from package_upload_entries where submission_id={submissionId} limit 1")
+
+  private val SQL_SELECT_PARTY_ALLOCATION_ENTRY =
+    SQL("select * from party_allocation_entries where submission_id={submissionId} limit 1")
```

Comment:
What's the reason for having `limit 1` in the query?

Reply:
just because there should only be one entry for a single submission and we only need one entry returned for confirmation of either accept or reject processing